### DISCUSSION TITLE
DataProducts for store and forward?

### DISCUSSION THREAD

**Discussion by zimri-leisher** [nasa/fprime#3076] [Q&A] [answered]
Hello,
I'm aware that DataProducts are intended to be used to downlink science/imagery/other misc. data. Would they be useful for downlinking stored telemetry data? Or is that more of a job for the BufferLogger/ComLogger?

**Comment by timcanham** [comment]
The roadmap has a notional ComLogger update that uses data products instead of just writing to a file.

**Reply by zimri-leisher** [reply]
Okay, so for now, you don't think DPs are the right tool for store and forward?
Would the next best thing in Svc be BufferLogger instead?

**Comment by timcanham** [comment, marked as answer]
In the past we've used ComLogger with known file names and transferred those files via FileDownlink.
